sources/packages/api/src/services/config/config.service.ts

```diff
@@ -8,7 +8,10 @@ export class ConfigService {
       auth: {
         url: process.env.KEYCLOAK_AUTH_URL,
         realm: process.env.KEYCLOAK_REALM,
-        clientId: process.env.KEYCLOAK_CLIENT,
+        clientIds: {
+          student: process.env.KEYCLOAK_CLIENT,
```

Comment:
Please consider renaming ENV variable KEYCLOAK_CLIENT to be more student explicit.

Reply:
Will do when we introduce new env variable